### ISSUE TITLE
Remove GitHub login

### ISSUE THREAD

**Issue by Pixel2016** [gitterHQ/sidecar#37]
Please how i put this to my web site . i want to remove git hub log in requirement.( so anybody can chat without register )  please answer asap.

**Comment by Pixel2016** [comment]
Please how i put this to my web site . i want to remove git hub log in requirement.( so anybody can chat without register )  please answer asap.

**Comment by MadLittleMods** [comment]
Sidecar is a glorified iframe of Gitter. Currently Gitter requires all people to be logged in to be able to chat in rooms which means that you need to sign in to be able to chat in room from Sidecar. We have talked about [adoptional ideas](https://medium.com/woah-basecamp-3/basecamp-3-adoptional-555e26a4b8b2#.mgw4jj1eg) but no plans yet.

tldr; not possible at the moment.
